test/functionalTest/cases/3176_entity_without_attributes_not_listed/entity_without_attributes_not_listed.test

```diff
@@ -0,0 +1,126 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+POST Entities V2 No Attributes
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+
+--SHELL--
+#
+# 01. POST /v2/registrations
+# 02. POST /v2/entities
+# 03. GET /v2/entities
+#
+
+
+echo "01. POST /v2/registrations"
+echo "=========================="
+payload='{
+  "id": "5aeb13827d4ef10a12a02630",
```

Comment:
`id` must not be provided in the registration creation operation. In fact, maybe Orion should report an error in this case (although that error checking is out of the scope of this PR)

Reply:
I just took the test right from the issue. This is how it came ...
But of course, the "id" inside the payload ... that should provoke an error?
Perhaps an issue is in place

Reply:
I think the issue is this one: https://github.com/telefonicaid/fiware-orion/issues/1256

Anyway `id` should be removed in this PR.

Reply:
Fixed in 3ed1863c8